<?xml version="1.0"?>
<!-- 此文件单独测试assigner使用 -->
<launch>
<node pkg="new_detector" type="new_assigner_single_dwa.py" name="assigner" output="screen">
  <param name="map_topic" value="/robot_1/move_base_node/global_costmap/costmap"/>
  <param name="global_frame" value="robot_1/map"/>
  <param name="info_radius" value="1"/> 
  <param name="info_multiplier" value="3.0"/> 
  <param name="hysteresis_radius" value="3.0"/> 
  <param name="hysteresis_gain" value="2.0"/> 
  <param name="frontiers_topic" value="filtered_points"/> 
  <param name="n_robots" value="1"/>
  <param name="namespace" value="robot_"/>
  <param name="delay_after_assignement" value="0.2"/>
  <param name="rate" value="100"/>
  </node>

</launch>


